core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.component.ts

```diff
@@ -86,6 +87,7 @@ export class EditThumbnailModalComponent implements OnInit {
       height: Math.round(height),
       width: Math.round(width)
     };
+    this.changeDetectorRef.detectChanges();
```

Comment:
Why do we now need `ChangeDetectorRef`?

Reply:
It's needed because still the parent is in AngularJS, and due to which change detection can be only achieved by manually applying it.